app/components/time_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= content_tag(
+      :'lg-time',
+      formatted_time,
+      **tag_options,
+      data: {
+        timestamp: time.iso8601,
```

Comment:
would it make dealing with timezones any easier if we serialized as seconds (or milliseconds) since the unix epoch instead of iso8601?

Reply:
> would it make dealing with timezones any easier if we serialized as seconds (or milliseconds) since the unix epoch instead of iso8601?

I don't know that it would make it easier or harder. Both could work just as well, unless I'm missing something.

Reply:
Leaving as iso8601 means we don't have to deal with seconds (Ruby) vs milliseconds (JS) so probably best to keep as-is